Crtx 240613 cooc azure to xsoar (#44708)
demisto/content: Packs/Azure/ReleaseNotes/1_2_0.md

#### Integrations

##### Azure

> - The **Test** button now validates the connection for the Client Credentials, Authorization Code, and Azure Managed Identities authentication types. For the Authorization Code flow, provide the Authorization code parameter before clicking **Test**.

Reviewer: ```suggestion
- Updated the **Test** button to validate the connection for the Client Credentials, Authorization Code, and Azure Managed Identities authentication types. For the Authorization Code flow, provide the *Authorization code* parameter before clicking **Test**.  
    The **Test** button is not supported for the Device Code authentication type. In this case, validate the connection by running the ***azure-auth-test*** command after executing the ***azure-auth-start*** and ***azure-auth-complete*** commands.

```

> - For the Device Code authentication type, validate the connection by running the *azure-auth-test* command after completing the *azure-auth-start* and *azure-auth-complete* steps. The **Test** button is not supported for this flow.

Reviewer: ```suggestion
```

#### Integrations

##### Azure

- Updated the **Test** button to validate the connection for the Client Credentials, Authorization Code, and Azure Managed Identities authentication types. For the Authorization Code flow, provide the *Authorization code* parameter before clicking **Test**.
- Updated the **Test** button for the Azure Managed Identities authentication type to validate the connection using a lightweight resource groups request.
- Added validation of the required parameters per authentication type before running a command.

> - Updated documentation and metadata.

Reviewer 2: ```suggestion
- Added support for installing the **Azure** content pack in the **Cortex XSOAR** and **Cortex XSIAM** Marketplaces (with version < 3.0), including a connection validation on **Test** for the Client Credentials, Authorization Code, and Azure Managed Identities authentication types (for Authorization Code, provide the Authorization code parameter before clicking Test; for Azure Managed Identities, the connection is validated using a lightweight resource groups request).
```
The first 2 points are a part of the additional support for the XSOAR and XSIAM legacy marketplaces.
The other 2 points are not visible to customers, there is no need to mention them.
